webdriver/tests/bidi/browsing_context/close/invalid.py

```diff
@@ -15,6 +15,12 @@ async def test_params_context_invalid_value(bidi_session):
         await bidi_session.browsing_context.close(context="foo")
 
 
+@pytest.mark.parametrize("value", ["", 42, {}, []])
```

Comment:
Maybe lets move this down to the end so that we have all the `context` checks next to each other.